cvat/apps/iam/schema.py

```diff
@@ -107,35 +78,6 @@ def get_security_definition(self, auto_schema):
         return [sessionid_schema, csrftoken_schema]
 
 class CustomAutoSchema(AutoSchema):
```

Comment:
This class can be moved to engine if there are no IAM specifics added. The only reason it is here is that we can't define it in the engine and import here.

Reply:
Moving this class in apps/engine/schema.py still leads to error due to [this line](https://github.com/opencv/cvat/blob/ed3dbe8f9d711a9894b51977b3ad11f4ec89f374/cvat/apps/engine/__init__.py#L7)